src/handlers/ui-events.js

```diff
@@ -469,4 +471,30 @@ export default {
   handleActivePlayerButtonClick() {
     this.openDialogView(dialogView.ONLINE_PEERS)
   },
+
+  /**
+   * @param {string} seed
+   */
+  handleRNGSeedChange(seed) {
+    const { origin, pathname, search, hash } = window.location
+    const queryParams = new URLSearchParams(search)
+
+    if (seed === '') {
```

Comment:
what do you think about doing `seed.trim()` here? or does it make sense to allow a seed that's empty spaces?

Reply:
Oh great catch! I think trimming here is a smart idea. Fixed in https://github.com/jeremyckahn/farmhand/commit/933a857e8b0d28d7e0b61e0d2f922fbbed856a1b!